### PR TITLE
Do not show validation errors in comparison modal

### LINKED ISSUE #354: Remove validation messages from process version comparison dialog

### PR DESCRIPTION
**Pull request by PPiedel**
This fixes #354

### CODE REVIEW

File: ui/client/components/Fields.js [hunk 1]
```diff
@@ -11,21 +11,20 @@ import ValidationLabels from "./modals/ValidationLabels";
...
     const opacity = this.props.isDragging ? 0 : 1;
+    const {index, field, isComparison} = this.props
 
     const validators = [notEmptyValidator];
     return this.props.connectDropTarget(this.props.connectDragSource(
       <div className="node-row movable-row" style={{opacity}}>
         <img src={dragHandleIcon} />
         <div className={"node-value fieldName" + markedClass}>
-          <input className={allValid(validators, field.name) ? "node-input" : "node-input node-input-with-error"}
+          <input className={isComparison || allValid(validators, field.name) ? "node-input" : "node-input node-input-with-error"}
```
Comment (mproch) [inline ui/client/components/Fields.js]: maybe let's name it showValidation? We can reuse it in the future :)

Comment (PPiedel) [reply ui/client/components/Fields.js]: Done

File: ui/client/components/graph/ExpressionSuggest.js [hunk 2]
```diff
@@ -103,14 +103,15 @@ class ExpressionSuggest extends React.Component {
   render() {
     const {isMarked} = this.props
     if (this.props.dataResolved) {
+      const isComparison = this.props.isComparison;
```
[the remark this answers is not in the document]
Comment (PPiedel) [reply ui/client/components/graph/ExpressionSuggest.js]: Done

File: ui/client/components/graph/EdgeDetailsModal.js [hunk 3]
```diff
@@ -117,6 +118,7 @@ class EdgeDetailsModal extends React.Component {
                     updateEdgeProp={this.updateEdgeProp}
                     readOnly={false}
                     edge={this.state.editedEdge}
+                    showValidation={false}
```
Comment (mproch) [inline ui/client/components/graph/EdgeDetailsModal.js]: I think it should be true here, and false in CompareVersionsDialog.js :D

**Comment by mproch** [comment]
@PPiedel I approved, but it's not ready yet...
In ExpressionSuggest (e.g. in filter) errors are still shown...

**Comment by mproch** [comment]
It's fine, sorry, I checked out wrong version....

**Pull request merged**